### PR TITLE
HL-298 & HL-301 & HL-302

### PR DESCRIPTION
**Pull request by JasminFarhi**
[Link to ticket](https://dltx.atlassian.net/jira/software/projects/HL/boards/82/backlog?selectedIssue=HL-298)

- Add styling for phone view
- If not intuitive from the ticket -> short summary of code changes
- Include screenshots for visual UI changes
Screen Shot 2022-12-12 at 3 02 47 pm
Screen Shot 2022-12-12 at 3 06 43 pm
Screen Shot 2022-12-12 at 3 06 56 pm

### CODE REVIEW

File: src/components/Bets/BetFilterGroup.tsx [hunk 1]
```diff
@@ -26,10 +26,13 @@ export const BetFilterGroup: React.FC<Props> = ({
         onClick={() => {
           onChange(key);
         }}
-        className={classNames("bg-white rounded px-2 shadow w-28", {
-          "bg-blue-500": key === currentOption,
-          "disabled:opacity-75": key !== currentOption
-        })}
+        className={classNames(
+          "bg-white rounded lg:px-2 shadow lg:w-28 w-13 px-1",
```
Comment (brettvniekerk) [inline src/components/Bets/BetFilterGroup.tsx:30]: you'll want to put your screenbreaks in left-to-right, i.e. `px-1 lg:px-2`

File: src/components/Card.tsx [hunk 2]
```diff
@@ -7,9 +7,9 @@ type Props = {
 };
 
 export const Card: React.FC<Props> = ({ title, data }) => (
-  <div className="px-4 py-5 bg-white shadow rounded-lg overflow-hidden sm:p-6 w-full">
-    <dt className="text-sm font-medium text-gray-500 truncate">{title}</dt>
-    <dd className="mt-1 text-3xl font-semibold text-gray-900">
+  <div className="px-2 py-1 bg-white shadow rounded-lg overflow-hidden sm:p-6 w-full">
+    <dt className="lg:text-sx font-medium text-gray-500 ">{title}</dt>
```
Comment (brettvniekerk) [inline src/components/Card.tsx:11]: `lg:text-xs` ?

**Review by brettvniekerk — APPROVED** [review]
Nice, two quick things

File: src/components/Bets/BetFilterGroup.tsx (hunk 1 above)
Comment (brettvniekerk) [reply src/components/Bets/BetFilterGroup.tsx:30]: helps with readability

File: src/components/Card.tsx (hunk 2 above)
Comment (OptimusOpus) [reply src/components/Card.tsx:11]: Yeah I think `lg:text-sx` is a typo?

File: src/components/Bets/BetFilterGroup.tsx [hunk 3]
```diff
@@ -26,10 +26,13 @@ export const BetFilterGroup: React.FC<Props> = ({
         onClick={() => {
           onChange(key);
         }}
-        className={classNames("bg-white rounded px-2 shadow w-28", {
-          "bg-blue-500": key === currentOption,
-          "disabled:opacity-75": key !== currentOption
-        })}
+        className={classNames(
+          "bg-white rounded px-1 lg:px-2 shadow lg:w-28 w-13",
```
Comment (brettvniekerk) [inline src/components/Bets/BetFilterGroup.tsx:30]: `w-13 lg:w-28`

Comment (pondpiu) [inline src/components/Bets/BetFilterGroup.tsx:30]: same thing as @brettvniekerk mention
put base css before one with breakpoint (w-13 before lg:w-28)

Comment (brettvniekerk) [reply src/components/Bets/BetFilterGroup.tsx:30]: this is also an issue in other files - really quick fix tho

File: src/components/Card.tsx (hunk 2 above)
Comment (JasminFarhi) [reply src/components/Card.tsx]: Yeah my bad, just removing it now. :)

**Review by OptimusOpus — APPROVED** [review]

**Review by pondpiu — APPROVED** [review]

**Review by bitcoinbrisbane — APPROVED** [review]

**Pull request merged**